src/components/EnterpriseList/index.jsx

```diff
@@ -12,6 +12,8 @@ import SearchBar from '../SearchBar';
 import SurveyPage from '../SurveyPage';
 import { updateUrl } from '../../utils';
 
+export const TITLE = 'Enterprise List';
```

Comment:
This title has annoyed me for the longest time. Can we change it to "My Enterprises" 😉